### DISCUSSION TITLE
Update the Netlify Template in the Repository

### DISCUSSION THREAD

**Discussion by nickytonline** [remix-run/remix#3092] [Proposals]
## Description

The Netlify template currently in the repository is due for an update. We currently have the https://github.com/netlify/remix-edge-template repository for the Netlify Edge functions template that we're looking to use as part of this update.

I'm going to chat with @ascorbic and will provide more details, but the gist of it is what's mentioned above.

@mcansh, I imagine once we've upgraded the template in the `main` branch, we can decommission the `experimental-netlify-edge` branch?

And as you mentioned, I should base my work on the `main` branch when updating a template, unlike regular PRs which are based on the `dev` branch correct?

**Comment by mcansh** [comment]
hey @nickytonline yep, both of those are correct :) - you could re-use the `experimental-netlify-edge` branch if you want, i think there's a few things we want to bring over (and a few things we'll need to undo)

**Comment by nickytonline** [comment]
What's easiest on your end? Get this merged into the `experimental-netlify-edge` branch first, and then we bring it into the main branch with "the few things we want to bring over (and a few things we'll need to undo)"?

**Reply by mcansh** [reply]
thought about this more and the `experimental-netlify-edge` branch is both for the adapter and a template, so actually i would leave it pointed to dev.

if the template there is good with a couple tweaks, it might be easier to just undo the couple commits specific for that branch's custom releases on the `experimental-netlify-edge` branch and merge it into dev which i think are just

- https://github.com/remix-run/remix/commit/5f7f6655797882b751201c97d76a3324688dfb83
- https://github.com/remix-run/remix/commit/243a414651f3f7772ccc02acf005140e38754919
- https://github.com/remix-run/remix/commit/40464a28b0f7f36ea39460e2fbd64164d3b59cb6
- https://github.com/remix-run/remix/commit/8eee27fc10a71d39765460b48da7a6cd0db06bb3
- https://github.com/remix-run/remix/commit/b345bdf5f62f07fd2cd01887401fd8219c48d766
